app/controllers/api/v1/responses_controller.rb

```diff
@@ -61,10 +61,21 @@ def see_results
         render json: { error: msg }, status: :unprocessable_entity
       end
 
+      def sign_out_user
+        ReminderEmailWorker.new(current_user, @response, TimePeriod.current).run_notification
```

Comment:
Is this supposed to be run only when "Save Draft" is clicked? Or all logging outs?

Reply:
This is executed for any  logging outs.